code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -161,6 +161,8 @@
 			see_in_dark = 4
 
 /mob/living/silicon/robot/proc/process_killswitch()
+	if(scrambledcodes)
```

Comment:
this will mess emagged traitor borgs snowflake to_chat()

Reply:
emagged traitor borgs don't get scrambled codes until self_destruct is called, then they unlink themselves. This is so things like nukeops borgs ignore everything

Reply:
Oh, that's true. They don't get scrambledcodes to avoid metatests.